kernel/src/constants.rs

```diff
@@ -1 +1,2 @@
-pub const VERSION: &str = "development";
+// pub const VERSION: &str = "development";
+pub const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
We should put it back to `pub const VERSION: &str = "development";` as it is what the build system is expecting to inject the `sha` at build time.

Reply:
Oh, sorry, it seems I forgot to push that change! My bad, I thought it was already reverted. Let me fix it. Thanks for catching it!

Reply:
Already pushed the change. It should be fine now.